crowbar_framework/app/models/node_object.rb

```diff
@@ -789,7 +789,7 @@ def save
     # Chef::Mixin::DeepMerge::deep_merge doesn't do
     def _remove_elements_from_node(old, new, from_node)
       old.each_key do |k|
-        if not new.has_key?(k)
+        if not new.key?(k)
```

Comment:
Style/Not: Use `!` instead of `not`. (https://github.com/bbatsov/ruby-style-guide#bang-not-not)